addresses.md

```diff
@@ -24,7 +24,31 @@ Here is an example resulting from a search for the text, `30 W 26th street, New
 }
 ```
 
-With an accuracy of point and an exact match, the confidence score is closer to 1. The confidence value decreases when centroid accuracy and a fallback match occurs. When that happens, multiple results may be returned so you can choose the one you intended.
+With an accuracy of point and an exact match, the confidence score is closer to 1. You also likely see higher scores with interpolated matches, but the confidence value decreases when centroid accuracy and a fallback match occurs. When that happens, multiple results may be returned so you can choose the one you intended.
+
+## Address interpolation
+
+When you search for an address and and there is not a precise match, interpolation occurs if there are enough known address values in that area. OpenAddresses and OpenStreetMap, which are the primary sources of addresses in the geocoder, provide locations for hundreds of millions of places globally. These sources build a framework for the interpolation, or estimation, of address numbers in areas where there is incomplete data.
```

Comment:
We should also mention US TIGER range data here, since that also drastically improves our US results in areas wthout OA coverage.